lib/rules/radix.js

```diff
@@ -6,67 +6,159 @@
 "use strict";
 
 //------------------------------------------------------------------------------
-// Rule Definition
+// Helpers
 //------------------------------------------------------------------------------
 
-module.exports = function(context) {
+var MODE_ALWAYS = "always",
+    MODE_AS_NEEDED = "as-needed";
 
-    var MODE_ALWAYS = "always",
-        MODE_AS_NEEDED = "as-needed";
+/**
+ * Checks whether a given variable is shadowed or not.
+ *
+ * @param {escope.Variable} variable - A variable to check.
+ * @returns {boolean} `true` if the variable is shadowed.
+ */
+function isShadowed(variable) {
+    return variable.defs.length >= 1;
+}
 
-    var mode = context.options[0] || MODE_ALWAYS;
+/**
+ * Checks whether a given node is a callee.
```

Comment:
Ah, yeah, I had forgotten it: https://github.com/eslint/eslint/blob/master/lib/ast-utils.js#L205